frontend/src/components/campAdmin/DialogActivityProgressLabelCreate.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <dialog-form
+    v-model="showDialog"
+    :loading="loading"
+    :error="error"
+    icon="mdi-package-variant"
+    :title="$tc('components.campAdmin.dialogActivityProgressLabelCreate.title')"
+    :submit-action="createDialogActivityProgressLabelCreate"
+    submit-color="success"
+    :cancel-action="close"
+  >
+    <template #activator="scope">
+      <slot name="activator" v-bind="scope" />
+    </template>
+
+    <dialog-activity-progress-label-form :activity-progress-label="entityData" />
+  </dialog-form>
+</template>
+
+<script>
+import DialogForm from '@/components/dialog/DialogForm.vue'
+import DialogBase from '@/components/dialog/DialogBase.vue'
+import DialogActivityProgressLabelForm from './DialogActivityProgressLabelForm.vue'
+
+export default {
+  name: 'DialogActivityProgressLabelCreate',
+  components: { DialogForm, DialogActivityProgressLabelForm },
+  extends: DialogBase,
+  props: {
+    camp: { type: Object, required: true },
+  },
+  data() {
+    return {
+      entityProperties: ['camp', 'position', 'title'],
+      entityUri: '/activity_progress_labels',
+    }
+  },
+  watch: {
+    showDialog: function (showDialog) {
+      if (showDialog) {
+        const labels = this.camp.progressLabels().items
+        const maxPosition = Math.max(0, ...labels.map((l) => l.position))
+
+        this.setEntityData({
+          camp: this.camp._meta.self,
+          position: maxPosition + 1,
```

Comment:
this must also be handled on the api side.
If 2 people add a label at the end, this wont end well